### PR TITLE
added translations of functional-api-guide.md

### PR DESCRIPTION
**Pull request by miyamotok0105**
functional-api-guide.mdの翻訳を行いました。なるべく文言を統一しました。確認お願いします。

### CODE REVIEW

File: sources/getting-started/functional-api-guide.md [hunk 1]
```diff
@@ -13,25 +13,25 @@ functional APIは，複数の出力があるモデルや有向非巡回グラフ
 下記のネットワークは`Sequential` modelによっても定義可能ですが，
 functional APIを使ったシンプルな例を見てきましょう．
 
-- layerのインスタンスは関数呼び出し可能で，戻り値としてtensorを返します
-- `Model`を定義することで入力と出力のtensorは接続されます
-- 上記で定義したモデルは`Sequential`と同様に利用可能です
+- layerのインスタンスは関数呼び出し可能で，戻り値としてtensorを返します．
```
Comment (nzw0301) [inline sources/getting-started/functional-api-guide.md:16]: - ここの箇条書きですが，`．`はなくても良いような気がしますが，どうでしょうか．
- tensor → テンソル

**Comment by nzw0301** [comment]
全体的なコメント
- 、 →，
- 。→．

File: sources/getting-started/functional-api-guide.md [hunk 2]
```diff
@@ -87,48 +87,48 @@ main inputはヘッドラインを整数のシーケンス(それぞれの整数
...
 x = Embedding(output_dim=512, input_dim=10000, input_length=100)(main_input)
 
-# a LSTM will transform the vector sequence into a single vector,
+# A LSTM will transform the vector sequence into a single vector,
 # containing information about the entire sequence
 lstm_out = LSTM(32)(x)
 ```
 
-補助損失を追加し，LSTMとEmbeddeding layerの学習をスムーズに行えるようにします．
+ここでは補助損失を挿入し、LSTMとEmbeddedingレイヤーをスムーズにトレーニングできるようにしますが、モデルでは主損失がはるかに高くなります。
```
Comment (nzw0301) [inline sources/getting-started/functional-api-guide.md:106]: - lossは挿入するとは言わない気がするので，`追加し`のままが良いかと思います
- `トレーニング`ではなく，`学習`のままが良いかと思います

**Comment by nzw0301** [comment]
全体的なコメント
- 、 →，
- 。→．

**Comment by nzw0301** [comment]
L419で
> (e.g. "what sport is the boy playing?" -> "football")

の訳を追加していただけますでしょうか

**Comment by Wakamatz** [comment]
L74

> Using the main loss function earlier in a model is a good regularization mechanism for deep models.

の訳を追加していただけますか？

File: sources/getting-started/functional-api-guide.md [hunk 3]
```diff
@@ -370,46 +371,46 @@ classification_model = Model([digit_a, digit_b], out)
...
 from keras.models import Model, Sequential
 
-# first, let's define a vision model using a Sequential model.
-# this model will encode an image into a vector.
+# First, let's define a vision model using a Sequential model.
+# This model will encode an image into a vector.
 vision_model = Sequential()
-vision_model.add(Convolution2D(64, 3, 3, activation='relu', border_mode='same', input_shape=(3, 224, 224)))
-vision_model.add(Convolution2D(64, 3, 3, activation='relu'))
+vision_model.add(Conv2D(64, (3, 3) activation='relu', padding='same', input_shape=(3, 224, 224)))
```
Comment (nzw0301) [inline sources/getting-started/functional-api-guide.md:381]: `(3, 3)`の後ろに`,`の追加をお願いします
